extensions/resource-deployment/src/services/bigDataClusterDeploymentProfile.ts

```diff
@@ -139,7 +139,7 @@ export class BigDataClusterDeploymentProfile {
 	}
 
 	public set controllerDataStorageSize(value: number) {
-		this._controlConfig.spec.storage.data.size = value;
+		this._controlConfig.spec.storage.data.size = `${value}Gi`;
```

Comment:
this is not for display, this is the what kubernetes expects. and it will be in the bdc.json and control.json. for display we are using localized "GB"